diagnose_customer_currency.py

```
@@ -0,0 +1,143 @@
+                    print(f"      Status: {sub.status}")
+                    print(f"      Currency: {sub.currency.upper()}")
+
+                    if sub.items.data:
+                        for item in sub.items.data:
+                            price = item.price
+                            print(f"      Price: {price.id}")
+                            print(f"      Amount: {price.unit_amount / 100:.2f} {price.currency.upper()}")
+
+                    # Check if it's EUR and active/trialing
+                    if sub.currency.lower() == 'eur' and sub.status in ['active', 'trialing', 'past_due']:
+                        print(f"\n      ⚠️  PROBLEM: Active EUR subscription!")
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)